extensions/amp-analytics/0.1/vendors/parsely.js

```diff
@@ -17,20 +17,26 @@
 import {jsonLiteral} from '../../../../src/json';
 
 const PARSELY_CONFIG = jsonLiteral({
+  'vars': {
+    'data': '{}',
+  },
   'requests': {
     'host': 'https://srv.pixel.parsely.com',
     'basePrefix':
       '${host}/plogger/?' +
       'rand=${timestamp}&' +
+      'plid=${pageViewId}&' +
       'idsite=${apikey}&' +
       'url=${ampdocUrl}&' +
       'urlref=${documentReferrer}&' +
       'screen=${screenWidth}x${screenHeight}%7C' +
       '${availableScreenWidth}x${availableScreenHeight}%7C' +
       '${screenColorDepth}&' +
+      'data=${data}&' +
       'title=${title}&' +
       'date=${timestamp}&' +
-      'ampid=${clientId(_parsely_visitor)}',
+      'ampid=${clientId(_parsely_visitor)}&' +
+      'pvid=${pageViewId}',
```

Comment:
Note that this change explicitly assumes that all AMP pageloads can only send a single pageview. Furthermore, it means the `plid` and `pvid` for any AMP event will always be the same. While that's not how the normal Parse.ly tracker works, I don't think it should cause any problems. @rachelannelise does that approach seem OK to you?

Reply:
I don't remember how the `plid` and `pvid` work in the traditional tracker but because you're adding them to the template that's used for both the heartbeats and the pageview those, along with `data`, will be sent on pageviews and heartbeats. If that's not the desired behavior you can move any of them down into the pageview or heartbeat templates.

Reply:
Yep, the intent here was to include those values on all events (both `pageviews` and `heartbeats`), as the values are used to group those events together in DPL queries: https://www.parse.ly/help/rawdata/schema#id-fields